.github/workflows/tests.yaml

```diff
@@ -2,11 +2,24 @@ name: Tests
 
 on:
   push:
+  schedule:
+  - cron:  '0 5 * * *'
```

Comment:
(forgot the comment in the initial review): how easy will it be to receive some sort of notification within slack about failures? Not sure whether we can just re-use the existing config or anything else, but it would be good to have this at some point (not within this PR though)

Reply:
We can use this code

https://github.com/stackrox/stackrox/blob/6e9a9ae54c358acfb33bf5d4ebc55d41a62d6675/.github/workflows/style.yaml#L117-L134